includes/class-custom-post-type-fund.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * The core plugin class file
+ *
+ * Defines the functions necessary to register our custom post types with
+ * WordPress.
+ *
+ * @link       https://premium.wpmudev.org/blog/create-wordpress-custom-post-types/
+ * @since      1.0.0
+ *
+ * @package    WSUWP_Plugin_iDonate_Post_Type_Fund
+ * @author     Blair Lierman
+ */
+class WSUWP_Plugin_iDonate_Post_Type_Fund {
+
+	/**
+	 * Initializes the plugin by registering the hooks necessary
+	 * for creating our custom post type within WordPress.
+	 *
+	 * @since    1.0.0
+	 */
+	public function init() {
+
+		add_action( 'init', array( $this, 'custom_post_types' ) );
+
+	}
+
+	function custom_post_types() {
+		$labels = array(
+			'name'				=> 'Funds',
+			'singular_name'		=> 'Fund',
+			'menu_name'			=> 'Funds',
+		);
+
+		$args = array(
```

Comment:
My *guess* is that we'll one day want to have `show_in_rest` as an argument here so that the funds can be exposed through the REST API. This will make them easier to use on sites other than the central Foundation site once the shortcode has been fully developed.

This isn't critical now, but something to think about as things move forward.